xmeffects.js

```diff
@@ -287,3 +293,4 @@ var effects_t1 = [  // effect functions on tick 1+
   eff_unimplemented,  // y
   eff_unimplemented   // z
 ];
+})(window || {});
```

Comment:
Good point. I actually meant just removing the || {} from window since we'd be overriding it, but this is fine...